### DISCUSSION TITLE
Improve the display of tags for media items

### DISCUSSION THREAD

**Discussion by sathyapriya91** [publishpress/publishpress-taxonomies#1527] [Feature Requests]
Have few images in media library
1. Select a image and add a tag
2. Go to 'Media Tags' and Edit the Tags name by Edit/Quick Edit
3. Check the image for the updated Tag name

The updated tag name should be displayed. 

Given "Name" field should appear in the site. Instead it displays the name of the slug. Update the slug name,the site displays the updated 'slug' field instead of the 'Name' field.

![Editied tag name](https://user-images.githubusercontent.com/83974581/121243519-a5d4eb00-c852-11eb-94b7-afd2657a348c.JPG)
![tag while creating](https://user-images.githubusercontent.com/83974581/121243523-a66d8180-c852-11eb-9f13-7bba3390debe.JPG)

**Comment by olatechpro** [comment]
Hi @stevejburge did you confirm if this is our issue and not wordpress behaviour for any media tags ?

**Comment by olatechpro** [comment]
When i check, in grid mode, wordpress display media terms by slug while in list edit mode, they display by name.

So, can't really see our bug here or something out of it natural way unless i'm missing something.

You can test this with any other tags aside media tags

**Comment by sathyapriya91** [comment]
@olatechpro  This scenario works well with posts and products tags. They displays the updated tag name in the corresponding posts and products. 

But the Media displays the Slug name instead of the name field.

**Comment by olatechpro** [comment]
Hi @sathyapriya91, Media is not our creation but WordPress. We only enable taxonomy support for media.
Right now, wordpress display any associated taxonomy be it (category, tags, media tag etc) as slug when editing the media in grid mode.

That's not our custumization
